test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs

```diff
@@ -1,102 +1,142 @@
-﻿
-using global::System;
-using Tizen.NUI;
+﻿/*
+ * Copyright(c) 2022 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
 using Tizen.NUI.BaseComponents;
-using NUnit.Framework;
 
 namespace Tizen.NUI.Samples
 {
     using log = Tizen.Log;
     public class SubWindowTest : IExample
     {
-        string tag = "NUITEST";
-        Window mainWin;
-        Window subWin1;
-        Window subWin2;
-        Window subWin3;
-        Timer tm;
-        bool belowParent;
-        void Initialize()
+        private const string tag = "NUITEST";
+        private const string keyBack = "XF86Back";
+        private const string keyEsc = "Escape";
+        private const string key1 = "1";
+        private const string key2 = "2";
+        private const string key3 = "3";
+        private const string key4 = "4";
+        private const string key5 = "5";
+        private const string key6 = "6";
+        private const string key7 = "7";
+        private Window mainWin;
+        private Window subWinFirst;
+        private Window subWinSecond;
+        private Window subWinThird;
+        private Timer timer;
+        private bool belowParent;
+        private const int oneSecond = 1000;
+        private const int testSize = 100;
+        private const int testPosition = 100;
+        private const float testPointSize = 12.0f;
+
+        public void Activate()
+        {
+            Initialize();
+        }
+
+        public void Deactivate()
+        {
+        }
+
+        private void Initialize()
         {
             mainWin = NUIApplication.GetDefaultWindow();
             mainWin.KeyEvent += OnKeyEvent;
             mainWin.BackgroundColor = Color.Cyan;
-            mainWin.WindowSize = new Size2D(500, 500);
+            mainWin.WindowSize = new Size2D(5 * testSize, 5 * testSize);
             mainWin.TouchEvent += WinTouchEvent;
             belowParent = false;
 
-            TextLabel text = new TextLabel("Hello Tizen NUI World");
-            text.HorizontalAlignment = HorizontalAlignment.Center;
-            text.VerticalAlignment = VerticalAlignment.Center;
-            text.TextColor = Color.Blue;
-            text.PointSize = 12.0f;
-            text.HeightResizePolicy = ResizePolicyType.FillToParent;
-            text.WidthResizePolicy = ResizePolicyType.FillToParent;
+            TextLabel text = new TextLabel("Hello Tizen NUI World")
```

Comment:
I suggest name of this variable to testText, helloWorldLabel.

Reply:
Thank for review. I fixed it.